xbmc/network/ZeroconfBrowser.cpp

```diff
@@ -60,9 +60,7 @@ CZeroconfBrowser::CZeroconfBrowser():mp_crit_sec(new CCriticalSection),m_started
 #endif
   AddServiceType("_ftp._tcp.");
   AddServiceType("_webdav._tcp.");
-#ifdef HAS_FILESYSTEM_NFS
-  AddServiceType("_nfs._tcp.");  
-#endif// HAS_FILESYSTEM_NFS
+  AddServiceType("_nfs._tcp.");
```

Comment:
is this correct?

Reply:
This needs to be addonized as well

Reply:
see https://github.com/xbmc/xbmc/pull/13037